.github/workflows/ci-unsupported.yml

```diff
@@ -0,0 +1,51 @@
+# The purpose of this CI is to run tests in a platform where there is no
+# detected native package manager
+
+name: CI unsupported
+
+on:
+  pull_request:
+    paths-ignore:
+      - 'doc/**'
+      - '**.md'
+      - '**.rst'
+      - '**.txt'
+
+jobs:
+
+  build:
+    name: CI on unsupported Linux
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Install toolchain
+      uses: ada-actions/toolchain@ce2020
+      with:
+        distrib: community
+    
+    - name: Install Python 3.x (required for the testsuite)
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Remove identifying file from OS
+      run: sudo rm -f /etc/os-release
```

Comment:
This could have unexpected side effects.
If you want we can add a config key to disable distrib features.

Reply:
I will do this. It may be interesting even for normal users to disable using crates not coming from sources.